spec/decanter/decanter_core_spec.rb

```diff
@@ -396,6 +397,22 @@
       end
     end
 
+    context 'with present non-required args containing an empty value' do
+      let(:decanter) {
+        Class.new(Decanter::Base) do
+          input :name, :string
+          input :description, :string
+        end
+      }
+      let(:params) { { name: '', description: 'My Trip Description' } }
+      let(:desired_result) { { name: nil, description: 'My Trip Description' } }
+      it 'should omit missing values' do
```

Comment:
The wording here is tricky, I'd be inclined to say it's passing through the missing values rather than omitting them.